### PR TITLE
Issue 1028: Add more filters to My Workbench / Recent content.

### PR DESCRIPTION
**Pull request by kasperg**
This changes the fields and filters shown under the My Workbench / Recent content.

The following fields are added:
- Associated library
- Profile forename and surname (with fallback to username)

The following filters are added:
- Associated library
- Profile forname
- Profile surname

The change is implemented using Features Override. Due to the way Features Override works it is not necessary to add that module as
a dependency.

Platform ticket: http://platform.dandigbib.org/issues/1028.

This change requires that Organic Groups is patched through the following pull requests:
- ding2/ding_staff#3
- ding2/ding_page#11
- ding2/ding_news#11
- ding2/ding_library#11
- ding2/ding_groups#1
- ding2/ding_event#14
- ding2/ding_base#10

### CODE REVIEW

**Comment by kasperg** [comment]
A couple of screenshots.

My workbench
min workbench kobenhavns biblioteker 2015-03-04 14-37-29

Recent content
seneste indhold kobenhavns biblioteker 2015-03-04 14-37-56

File: ding_content.info [hunk 1]
```diff
@@ -5,36 +5,31 @@ package = Ding!
...
-dependencies[] = media_internet
 dependencies[] = media_vimeo
 dependencies[] = media_wysiwyg
 dependencies[] = media_youtube
 dependencies[] = menu
 dependencies[] = nodequeue
 dependencies[] = opening_hours
-dependencies[] = page_manager
-dependencies[] = panels
 dependencies[] = panels_node
```
Comment (cableman) [inline ding_content.info]: I'am not sure that all these dependencies should be removed.

**Comment by cableman** [comment]
Review completed: The patches to the dependent module have been merged. A single question to dependencies removed from the info has to be answered before merge.

File: ding_content.info (hunk 1 above)
Comment (kasperg) [inline ding_content.info]: You are right. I will readd them.

**Comment by kasperg** [comment]
@cableman: Updated.

**Comment by cableman** [comment]
Reviewed and approved

**Pull request merged**